test/functional/page_objects/home_page.js

```diff
@@ -63,6 +66,17 @@ export function HomePageProvider({ getService }) {
       await testSubjects.click(`launchSampleDataSet${id}`);
     }
 
+    async loadSavedObjects() {
+      // Button at bottom of page, move into view before clicking
+      const loadBtn = await testSubjects.find('loadSavedObjects');
+      await remote.moveMouseTo(loadBtn);
+      await testSubjects.click('loadSavedObjects');
+      await retry.try(async () => {
+        const successMsgExists = await await testSubjects.exists('loadSavedObjects_success');
```

Comment:
Two `awaits` here.  Also, have you tried just extending the timeout?  You can do something like:

`await testSubjects.exists('loadSavedObjects_success', 10000)` if the default timeout of 1 second is not long enough.

Another option is to just do await testSubjects.find('loadSavedObjects_success');

That should have an internal retry since it assumes it's supposed to exist:

```
    async _ensureElementWithTimeout(timeout, getElementFunction) {
      try {
        const remoteWithTimeout = remote.setFindTimeout(timeout);
        return await retry.try(async () => {
          const element = await getElementFunction(remoteWithTimeout);
          // Calling any method forces a staleness check
          element.isEnabled();
          return element;
        });
      } finally {
        remote.setFindTimeout(defaultFindTimeout);
      }
    }
```

Reply:
nm, I see you already got rid of the second await in a follow up commit.  

cc @spalger - I think this is a deficiency to having non-squashed commits.  How do you keep up with all the changes from commit to commit to commit?

For instance I reviewed before this commit.  I wanted to see the functional tests added so I went to that commit and added the comment.  But the next commit fixed it.  So what would your flow be?  Start at commit x and review each one, x -> x + 1 -> x + 2 ... ?  What if an x + n commit fixed something you commented on prior?  Just go back and delete it?

Or do you review all commits from one commit onward but not the pull merges?  Doesn't seem possible to do this in the git UI.  It's either review all changes, or review commit by commit.